kubernetes/types.go

```diff
@@ -1,10 +1,11 @@
 package kubernetes
 
 import (
+	"time"
+
 	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-	"time"
```

Comment:
This change was already done - see https://github.com/kiali/kiali/blob/master/kubernetes/types.go#L4-L8

Looks like this PR needs a rebase.